var/spack/repos/builtin/packages/lzo/package.py

```diff
@@ -37,13 +37,14 @@ class Lzo(Package):
     version('2.06', '95380bd4081f85ef08c5209f4107e9f8')
     version('2.05', 'c67cda5fa191bab761c7cb06fe091e36')
 
-    def install(self, spec, prefix):
-        configure_args = [
-            '--prefix={0}'.format(prefix),
+    def configure_args(self):
+        return [
             '--disable-dependency-tracking',
             '--enable-shared'
         ]
-        configure(*configure_args)
-        make()
+
+    @AutotoolsPackage.sanity_check('build')
+    @AutotoolsPackage.on_package_attributes(run_tests=True)
+    def check(self):
```

Comment:
I would say 2/3 of the Autotools packages I've worked with have had `make check` and maybe 1/3 or less have had `make installcheck`. Regardless, even if that target doesn't exist, I don't think `make('check')` crashes, so I would be in favor of making both of those the defaults for Autotools.